### PR TITLE
Update: added _lang property (fixes #470)

### LINKED ISSUE #470: Add _lang property
It should be possible to specify the language of individual content items for use with translation etc.

### PR DESCRIPTION
**Pull request by chris-steele**
#470

### CODE REVIEW

**Review by taylortom — CHANGES_REQUESTED** [review]

File: schema/content.schema.json [hunk 1]
```diff
@@ -10,7 +10,10 @@
     "_friendlyId": {
       "type": "string",
       "title": "Friendly id",
-      "description": "A human-readable ID for this content object which will replace the standard _id field on course build (useful in instances where the _id must be specified in the config of an extension). Must be unique to this course."
+      "description": "A human-readable ID for this content object which will replace the standard _id field on course build (useful in instances where the _id must be specified in the config of an extension). Must be unique to this course.",
+      "_backboneForms": {
+        "editorAttrs": { "disabled": true }
```
[the remark this answers is not in the document]
Comment (chris-steele) [reply schema/content.schema.json]: Amended

File: schema/content.schema.json [hunk 2]
```diff
@@ -25,6 +28,15 @@
         "editorOnly": true
       }
     },
+    "_lang": {
+      "type": "string",
+      "title": "Language",
+      "description": "The language/locale e.g. 'en' or 'fr-CA'.",
+      "default": "en",
+      "_backboneForms": {
+        "editorAttrs": { "disabled": true }
```
[the remark this answers is not in the document]
Comment (chris-steele) [reply schema/content.schema.json]: Amended

**Comment by chris-steele** [comment]
FYI this PR will likely be binned soon as we have concluded that `_lang` is not required.

**Comment by taylortom** [comment]
Nothing to see here...after some discussion it's been decided that this change isn't needed on the framework side.

**Pull request closed without merging**